### PR TITLE
BugFix which is for tsconfig seperation

### PR DESCRIPTION
**Pull request by bitPogo**

### CODE REVIEW

File: src/server/app.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
-import fs from 'fs';
+import * as fs from 'fs';
```
Comment (jakobw) [inline src/server/app.ts]: I don't think these changes are necessary.

I have the branch checked out, and this is fine: 
`git checkout master src/server/app.ts && docker-compose run --rm node npm run build-server`

Comment (bitPogo) [reply src/server/app.ts]: done

Comment (jakobw) [reply src/server/app.ts]: Sorry, I meant that none of the changes in `src/server/app.ts` are needed.

Comment (bitPogo) [reply src/server/app.ts]: done

**Pull request merged**